src/views/favorite_places.py

```diff
@@ -0,0 +1,36 @@
+from flask import Blueprint, request
+from flask_cors import CORS
+
+from controllers import favorite_places as controller
+from utils.formatters import create_response
+
+from utils.validators.general import validate_header, validate_token
+
+favorite_places_blueprint = Blueprint('favorite_places', __name__, url_prefix='/api')
+CORS(favorite_places_blueprint)
+
+
+@favorite_places_blueprint.route('/places/', methods=['POST'])
+@validate_header
+@validate_token
+def post_favorite_place(username):
+    response, status = controller.create_favorite_place(
+        request.json, username)
```

Comment:
In the function `controller.create_favorire_place`the order of the parameters is defined differently from what is being passed in the function call:
https://github.com/fga-eps-mds/2020.1-stay-safe-user-service/blob/857fbbf086e303de481d55fe2cfd95cc3001307d/src/controllers/favorite_places.py#L16

So when I try to post, I have this error:
![image](https://user-images.githubusercontent.com/30883179/97306468-4b47cb00-183d-11eb-928a-313daa37fb6a.png)